pkg/id/message.go

```diff
@@ -27,26 +23,8 @@ func GenerateMessageID() string {
 	uuidv7, err := defaultUUIDGenerator()
 	if err != nil {
 		slog.Debug("Issue generating uuidv7, using sha256 and timestamp instead", "error", err)
```

Comment:
Since this is a public library now we mustn't use slog since the consumer of the library may be using a different logging library. Can you remove this log line?